schemas/dc-library.xsd

```diff
@@ -94,6 +97,7 @@
       <xs:simpleContent>
         <xs:extension base="xs:string">
           <xs:attribute name="root" type="xs:string" use="optional"/>
+          <xs:attribute name="technical" type="xs:string" use="optional"/>
```

Comment:
Correct. I tagged the wrong one - citation, which is inside meta - should not allow technical, but cite and code-cite should. It looks like citation still has it.